contracts/Governance/AccessControlledV5.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.5.16;
+
+import "./Utils/InitializableV5.sol";
+import "./Utils/Ownable2StepUpgradeableV5.sol";
+import "./IAccessControlManagerV5.sol";
+
+/**
+ * @title Venus Access Control Contract.
+ * @dev This contract is helper between access control manager and actual contract
+ * This contract further inherited by other contract to integrate access controlled mechanism
+ * It provides initialise methods and verifying access methods
+ */
+
+contract AccessControlledV5 is InitializableV5, Ownable2StepUpgradeableV5 {
+    /// @notice Access control manager contract
+    IAccessControlManagerV5 private _accessControlManager;
+
+    /**
+     * @dev This empty reserved space is put in place to allow future versions to add new
+     * variables without shifting down storage in the inheritance chain.
+     * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
+     */
+    uint256[49] private __gap;
+
+    /// @notice Emitted when access control manager contract address is changed
+    event NewAccessControlManager(address oldAccessControlManager, address newAccessControlManager);
+
+    function __AccessControlled_init(address accessControlManager_) internal onlyInitializing {
+        __Ownable2Step_init();
+        __AccessControlled_init_unchained(accessControlManager_);
+    }
+
+    function __AccessControlled_init_unchained(address accessControlManager_) internal onlyInitializing {
+        _setAccessControlManager(accessControlManager_);
+    }
```

Comment:
We don't initialize this base contract in the vault contracts, so maybe we could simplify this V5 mixin by removing the initialization altogether? If we strip the initializers, we could remove some libraries we depend on as well.